test/views/TaxYearSummaryViewSpec.scala

```diff
@@ -508,12 +508,12 @@ class TaxYearSummaryViewSpec extends ViewSpec with FeatureSwitching {
         errorMessageList.child(27).text shouldBe messages("tax-year-summary.message.C55502")
         errorMessageList.child(28).text shouldBe messages("tax-year-summary.message.C55503")
         errorMessageList.child(29).text shouldBe messages("tax-year-summary.message.C55508")
-        errorMessageList.child(30).text shouldBe messages("tax-year-summary.message.C55008", dateService.getCurrentDate.toLongDate)
-        errorMessageList.child(31).text shouldBe messages("tax-year-summary.message.C55011", dateService.getCurrentDate.toLongDate)
+        errorMessageList.child(30).text shouldBe messages("tax-year-summary.message.C55008", dateService.getCurrentDate(isEnabled(TimeMachineAddYear)).toLongDate)
```

Comment:
might be easy to call it once and keep this state in the very beginning of this execution scope

Reply:
Have fixed